### DISCUSSION TITLE
Need a list of supported models

### DISCUSSION THREAD

**Discussion by AfaroX** [sdwebui-w-horde/sd-webui-stable-horde-worker#97] [Q&A]
Where can i find a list of all models supported by the worker? When i enable my worker extesion, all i see is "status: Current model [model name here] not found on StableHorde"

**Comment by MaikoTan** [comment]
Yep, this is definitely worth being a feature of our worker bridge, I just have no idea how to implement this.

**Comment by zba** [comment]
Yes, and where to download them, no one model is "supported" including basic stable diffusion, I would to run horde worker on my rtx , but have no idea where to get compatible models. It is pity that install working client is easer than worker. Also discord link from official site do not work, etc, I not understand how this project receives workers. 
all models do this, regardless if they "supported" or not in horde client

```
webui-docker-auto-1  | Calculating md5sum for dreamshaper_6BakedVae.safetensors
webui-docker-auto-1  | md5sum for dreamshaper_6BakedVae.safetensors is 370c40b66389081a0eeafe8078e7be2a  but it's not supported by StableHorde

```
